docs/constitution.rst

```diff
@@ -0,0 +1,60 @@
+Mu's Constitution
+=================
+
+Our Values
+----------
+
+Happily, Mu's users and developers are a diverse and international community
+who maintain a reputation as a friendly, welcoming and dynamic group.
+
+We trust that such collaborators will treat each other in a way that reflects
+the widely help view that diversity and friendliness are strengths of our
+community to be celebrated and fostered.
+
+We believe anyone involved in Mu's community has a right to:
+
+* be treated with courtesy, dignity and respect;
+* be free from any form of discrimination, victimisation, harassment or
+  bullying.
+* enjoy an environment free from unwelcome behaviour, inappropriate language
+  and unsuitable imagery.
+
+To this end, we hope participants in our community abide by the spirit of our
+:doc:`code_of_conduct`.
+
+The Role of Administrators
+--------------------------
+
+Contributing to Mu
+------------------
+
+Managing Expectations
+---------------------
+
+In Case of Problems
```

Comment:
I feel there is an strong relationship / intersection / overlap with the Code of Conduct, here. Could we fully / partially delegate this section to the CoC, by "linking" to it? Trying to DRY things up. :)

With that in mind:
* Note that Mu's Code of Conduct `./CODE_OF_CONDUCT.rst` says "We expect contributors to follow the Python Software Foundation's Code of Conduct, reproduced below" but the [linked PSF CoC](https://www.python.org/psf/conduct/) has different content from what is reproduced.
* "Problems", as in the section title, is a very complex word. What someone sees and feels as a problem, others might see differently. Maybe that's ok, I'm not sure (thus the relevance of CoCs).

Reply:
Hmmm.... I'll need to think about this.

I see the constitution and CoC complementing each other.